contracts/test/Summa.ts

```diff
@@ -255,113 +242,85 @@ describe("Summa Contract", () => {
       const calldata: any = JSON.parse(jsonData);
 
       mstRoot = calldata.public_inputs[0];
```

Comment:
We will no longer have `solvency_proof_solidity_calldata.json` and its generator script in the future. How do you think we should modify the flow accordingly? One suggestion that I have is to modify the `gen_inclusion_verifier.rs` example with `generate_mst_commitment.rs` that outputs something like `commitment_calldata.json`

Reply:
I think it's a good idea.